app/web/features/auth/phone/ChangePhone.tsx

```diff
@@ -80,7 +80,8 @@ export default function ChangePhone({
     handleSubmit: verifyHandleSubmit,
```

Comment:
Translations are off (not sure if introduces in this PR):

![Screenshot from 2024-12-12 13-14-45](https://github.com/user-attachments/assets/fa44c6a9-4a90-4c48-96cc-01748d2af377)
---
![Screenshot from 2024-12-12 13-39-30](https://github.com/user-attachments/assets/17e50793-2903-4345-a69d-34d8667503c0)
---
![Screenshot from 2024-12-12 13-39-48](https://github.com/user-attachments/assets/db091502-480e-408d-94d4-2a814161f7e8)

Reply:
I think fixed but I didn't donate so can't go all the way to the verification so please check me.

Translation fixed though.

Reply:
You can use the stripe test cards on next, 4242424242424242 will succeed for instance. Just fyi for the future!

Reply:
Ohhhh yes I forgot you can do that with Stripe, thanks!